tests/.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "trailingComma": "es5",
+  "printWidth": 100,
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "objectWrap": "collapse"
```

Comment:
Do you really need tabWidth and printWidth, shouldn't it pick them up from the editorconfig?